### ISSUE TITLE
send-money.sh fails at step #9

### ISSUE THREAD

**Issue by gacallea** [input-output-hk/jormungandr-qa#14]
Dear QA Team,

TL;DR: trying to move my own tADA from my pool's pledge address to my Yoroi ITN wallet, and it fails with the following error (the address is valid and generated by Yoroi):

```
##9. Check the account's balance
error: Invalid value for '<account-id>': account parameter 'addr1ssgltauckscq82a8t7ad0jc28ztx7g55893xj3pelly5fwaqkz744gsxtwtcudg840tjfh5dqs6xqa3sd04zddd3d4g2nsqcw5e4ss8fx5y7fu' isn't an account address
```

Longer story:

When I was creating my pool, I moved all my tADA to the pool's pledge address so I could register it on the registry. Now that I need to participate in the bootstrap initiative (helping small pool kickstart activity), I need to move them back to the wallet to partake in the necessary steps, but it fails.

This is the full command I run:

```
./send-money.sh addr1ssgltauckscq82a8t7ad0jc28ztx7g55893xj3pelly5fwaqkz744gsxtwtcudg840tjfh5dqs6xqa3sd04zddd3d4g2nsqcw5e4ss8fx5y7fu 532600000 9300 $(cat skkey)
```

Everything until the very last step works.

I would appreciate if you could help me move my only tADA. It would suck to have them stuck there forever.

Please and thank you

ps: the current balance for the above address was kindly donated by another pool operator who's in the bootstrap initiative. mine always fail moving.

**Comment by dorin100** [comment]
Dear @gacallea , 

The specified script is only an example and it does not work with addresses generated by Yoroi/Daedalus. 

Please use one of the wallets for finalizing the desired operation.

**Comment by gacallea** [comment]
Dear @dorin100 

thank you so much for the quick reply, I appreciate.

I would use Yoroi, but I don't know how associate the pledge address with it. Any pointers would be awesome.

Please and thank you

**Comment by dorin100** [comment]
I am not sure how you generated the pledged address but if you have a mnemonic, you would be able to restore the wallet in Yoroi or Daedalus. 

For similar questions feel free to open a ticket to our support team here - https://iohk.zendesk.com/hc/en-us

**Comment by gacallea** [comment]
@dorin100 

the address was created with QA very own https://github.com/input-output-hk/jormungandr-qa/blob/master/scripts/createAddress.sh

for complete info:

I followed IOHK's official guides to create/register the pool, which use QA scripts for any need.

I hope this can be helpful, otherwise I'll try zendesk.

Please and thank you

**Comment by dorin100** [comment]
@gacallea 

It seems that what you are trying to do is not possible neither with Daedalus nor with Yoroi so the best way to approach this would be through Support. 

**PS**: in order to have your funds in one of the wallets, most probably you will need to initiate a transaction from jcli like you already was trying (but adding small changes to the script). Please note that the script, in your case, is returning an error when trying to interrogate the destination address  because jcli does not know to interpret that type of address - basically, if your node is synced to the main chain after the transaction is processed, you should see the funds on your address (in explorer)

**Comment by gacallea** [comment]
@dorin100 I will contact Support, thank you kindly for your help :)
